.gitignore

```diff
@@ -40,3 +40,7 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+# Ignored files in order to avoid error in heroku build
+# yarn.lock
```

Comment:
Once those files were added to git, adding them to the .gitignore won't work as expected. Git will continue to track them. It is necessary to clean git cache.

Reply:
Cool! anyway, Heroku does not handle cache. In our local repo, it's necessary to do that. :)